tendrl/commons/objects/cluster/atoms/configure_collectd/__init__.py

```diff
@@ -0,0 +1,86 @@
+import etcd
+import gevent
+import uuid
+
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons import objects
+from tendrl.commons.objects import AtomExecutionFailedError
+from tendrl.commons.objects.job import Job
+
+
+class ConfigureCollectd(objects.BaseAtom):
```

Comment:
I dont think we need to trigger another job from this atom, simply merge the flow here https://github.com/Tendrl/node-agent/pull/552/files#diff-409adf868f9d4c01da99c6fa4a3cfcbfR1  

in this atom itself. This atom will be run once for every node being imported

The reason behind this is, the entire new mvp is based on monitoring and after some thought it seems we shouldnt decouple import + configure monitoring.